pkg/apis/etcd/v1beta2/cluster.go

```diff
@@ -206,6 +207,11 @@ func (c *ClusterSpec) Validate() error {
 				return errors.New("spec: pod labels contains reserved label")
 			}
 		}
+		if c.Pod.PV != nil {
+			if c.Pod.PV.VolumeSizeInMB < minPVSize {
```

Comment:
maybe we just set it back to min instead of erroring it? i forgot about how k8s resource limit works here for the similar case though. probably we want the consistency here.